packages/components/src/higher-order/with-focus-outside/index.js

```diff
@@ -79,6 +79,14 @@ export default createHigherOrderComponent(
 				}
 
 				this.blurCheckTimeout = setTimeout( () => {
+					// If document is not focused then focus should remain
+					// inside the wrapped component and therefore we cancel
+					// this blur event thereby leaving focus in place.
+					// https://developer.mozilla.org/en-US/docs/Web/API/Document/hasFocus.
+					if ( ! document.hasFocus() ) {
+						event.preventDefault();
```

Comment:
Is this line necessary, or is the fix contained in just avoiding the callback?

Notably, the `focus` event is not cancellable, so it doesn't seem like this should be doing anything.

See: https://developer.mozilla.org/en-US/docs/Web/API/Element/focus_event